server/services/stats/side/test.go

```diff
@@ -116,6 +119,20 @@ func (c *testGaClient) setQuestions(rows map[int64]*schema.AnalyticsGaRow) {
 	c.questions = rows
 }
 
+type testTsClient struct{}
+
+func NewTestTcClient() TcClient {
+	return &testTsClient{}
+}
+
+func (t *testTsClient) GetTCBuildInfo() (*TCBuildInfo, error) {
+	return &TCBuildInfo{
+		BuildStatuses:     []string{"SUCCESS", "FAILURE", "ERROR"},
```

Comment:
very strange struct

we have `slice` of build results and only author of last one.

but are you sure that slice sorted? and how know author of every else change?